core/tests/protractor.conf.js

```diff
@@ -187,3 +163,53 @@ exports.config = {
   // (0 if the tests passed or 1 if not).
   onCleanUp: function() {}
 };
+
+if (process.env.TRAVIS) {
+  // SAUCE_USERNAME(SauceLabs username) and SAUSE_ACCESS_KEY(SauceLabs Access Key)
+  // will be fetched as Travis env variable 
+  // https://support.saucelabs.com/customer/portal/articles/2029412-running-protractor-tests-on-sauce-labs
+  // https://support.saucelabs.com/customer/portal/articles/2005335-angular-js-protractor-example
+  config.sauceUser = process.env.SAUCE_USERNAME;
+  config.sauceKey = process.env.SAUCE_ACCESS_KEY;
+  config.capabilities = {
+    'name': 'oppia',
+    'browserName': 'chrome',
+    'seleniumVersion': '2.49.0',
+    'chromedriverVersion': '2.20',
+    'tunnel-identifier': process.env.TRAVIS_JOB_NUMBER,
+    'build': process.env.TRAVIS_BUILD_NUMBER,
+    'platform': 'OS X 10.10',
+    'screen-resolution': '2048x1536'
+  };
+  config.onPrepare = function() {
+    // Set a wide enough window size for the navbar in the gallery to display
+    // fully.
+    browser.driver.manage().window().setSize(1920, 1440);
```

Comment:
This is very very big. Does it need to be so big? It's just that it's not really typical of most monitor screen sizes (I think).